Fixed byline width and position.

soehnlein opened the pull request.

ehlertij commented on assets/stylesheets/main.css:
```diff
@@ -51,31 +51,28 @@ table {
-    height: 24px;
+  height: 24px;
 
-    background-color: white;
```
I would put your `background-color: white` back on here and also add a `width: 100%` style as well. This will make it push to the edges of the screen. Then you can make put `background-color: #ccc` on your `body` to make sure the background color extends across the entire page. Without that, you can get a little white strip on the bottom of the page if the window is big enough.

ehlertij commented:
I'd also consider adding a `min-width` onto your `.header` class to make sure it can never scrunch down entirely in narrower windows.

soehnlein commented:
I changed the `width` on `.header` to be `min-width: 67%`, added `background-color` to both `.body` and `.byline`. Didn't add `width: 100%` to the `.byline` because it made the `.byline` div bigger, and the whole page scroll horizontally.

ehlertij commented:
With the styles you have now, the `byline` section doesn't always expand all the way to the right edge of the window. After some googling, I think I figured out the best solution to this. Following this example: http://stackoverflow.com/questions/1260122/expand-div-to-take-remaining-width

* Get rid of `display: inline-block` on both `.title` and `.byline`.
* On `.byline` replace `position: absolute` with `overflow: hidden`. 

That should work for most window sizes. You might still want a `min-width` on `body`. I'll let you play with that and see what makes sense (if anything).

soehnlein commented:
Cool, I found that solution too. I changed what it said, but now the byline gets knocked down when the window is too small, which is what I don't want. I think we discussed a solution for this yesterday but I forgot what it was.

ehlertij commented:
I think a `min-width` on the body will do the trick.

soehnlein commented:
Done! Think it's good to merge?

ehlertij commented:
Fix your merge conflicts then, :ship: it!
